.github/workflows/app-publish.yaml

```diff
@@ -0,0 +1,73 @@
+name: App Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    environment: release
+    if: ${{ github.repository == 'jellyfin/jellyfin-android' }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
+      - name: Set JELLYFIN_VERSION
+        run: echo "JELLYFIN_VERSION=$(echo ${GITHUB_REF#refs/tags/v} | tr / -)" >> $GITHUB_ENV
+
+      - name: Build Signed Release
+        env:
+          KEYSTORE: ${{ secrets.KEYSTORE }}
+          KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
+          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
+          KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
```

Comment:
- [ ] These need to be added (preferably on organization level so we can re-use it in the ATV app later).

_(mark when added)_

Reply:
What are KEY_ALIAS and KEY_PASSWORD? I have added KEYSTORE and KEYSTORE_PASSWORD for this repo and Android TV.

Reply:
After further review, KEY_ALIAS has been added. I believe KEY_PASSWORD is unused here.